diff --git a/react-native-webassembly.podspec b/react-native-webassembly.podspec
index aba1f2fcaf442d6d5f8c2f18863055252c347a64..03bf3dbb2379139dbc429ec4b082f5b0c9145694 100644
--- a/react-native-webassembly.podspec
+++ b/react-native-webassembly.podspec
@@ -26,7 +26,7 @@ Pod::Spec.new do |s|
   }
   s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
   s.dependency "React-Codegen"
-  s.dependency "RCT-Folly"
+  s.dependency "React-Core"
   s.dependency "RCTRequired"
   s.dependency "RCTTypeSafety"
   s.dependency "ReactCommon/turbomodule/core"
